.travis.yml

```diff
@@ -0,0 +1,22 @@
+dist: bionic
+git:
+  submodules: false
+before_install:
+  - git clone https://${TOKEN}:x-oauth-basic@github.com/meedan/configurator ./configurator
+before_script:
+  - sed -i 's/git@github.com:/https:\/\/github.com\//' .gitmodules
+  - git submodule update --init --recursive
+  - sed -i 's/--abort-on-container-exit/-d/' bin/first-build.sh
```

Comment:
Would be good to keep the `--abort-on-container-exit`, this way the tests can fail right away if a container can't boot... unless there is another reason to remove it?